samples/Hangfire.Core.MvcApplication/Hangfire.Core.MvcApplication.csproj

```diff
@@ -10,12 +10,13 @@
     <PackageReference Include="Hangfire" Version="1.7.8" />
     <PackageReference Include="Hangfire.Heartbeat" Version="0.5.0" />
     <PackageReference Include="Hangfire.MemoryStorage.Core" Version="1.4.0" />
+    <PackageReference Include="Hangfire.PostgreSql" Version="1.6.4.2" />
     <PackageReference Include="Microsoft.AspNetCore.App" />
-    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
```

Comment:
this may be needed for the razor page code auto-generation (ie. TagsSearchPage.generated.cs)

Reply:
they moved to aspnetcore in 3.1.
but i'm rolling it back to 2.2